From 1b2964944e5182fd6b04626ef24678b5848a4af4 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Mon, 8 Nov 2021 11:46:18 +0100
Subject: [PATCH 091/529] ARM: dts: stm32: remove thermal passive trip point on
 stm32mp151

Remove thermal passive trip point because it is useless.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: I18a3b34c7cfc589b3e4f6aa2a03a4e0c0b65b157
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/228360
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/276992
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 arch/arm/boot/dts/stm32mp151.dtsi | 6 ------
 1 file changed, 6 deletions(-)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -92,12 +92,6 @@
 			thermal-sensors = <&dts>;
 
 			trips {
-				cpu_alert1: cpu-alert1 {
-					temperature = <85000>;
-					hysteresis = <0>;
-					type = "passive";
-				};
-
 				cpu-crit {
 					temperature = <120000>;
 					hysteresis = <0>;
